app/build.gradle

```diff
@@ -15,6 +15,8 @@ android {
         targetSdkVersion 30
         versionCode 1
         versionName "1.0"
+        renderscriptTargetApi 22
```

Comment:
@danacealfonso 
Please refrain from using third-party libraries without consultation. Check with all of the members first.

Reply:
Apologies for implementing libraries third-party libraries. Reverted back to commit f1cc47b9d4345d56493409bb9906022ec10412a7